### ISSUE TITLE
Add accessibilty info and link to footer

### ISSUE THREAD

**Issue by corylown** [sul-dlss/dlme#1819]
From Astrid:

Here are the accessibility footers for the English and Arabic versions of the DLME site:

English (left align): If you cannot access content or use features on this website due to a disability, please report your accessibility issue.
Arabic (right align): إذا لم تتمكن من الوصول إلى المحتوى أو استخدام ميزات هذا الموقع بسبب الإعاقة، يرجى الإبلاغ عن مشكلة إمكانية الوصول

The link in both cases goes to: [library-accessibility-contact@lists.stanford.edu](mailto:library-accessibility-contact@lists.stanford.edu)

Mockups (Since the footer is on that navy background, we can't use the digital blue link color because it won't have enough contrast so I underlined the link. Also, let me know if the alignment is an issue. I just noticed the logos are center aligned with the full width of the page. I'd like the accessibility line to be left/right aligned with the first/last logo)

<img width="1044" alt="image" src="https://github.com/sul-dlss/dlme/assets/458247/276fa321-30be-4833-8cc3-66c900abf93b">
